lib/Controller/StackApiController.php

```diff
@@ -39,18 +39,20 @@
  * @package OCA\Deck\Controller
  */
 class StackApiController extends ApiController {
-	private $boardService;
-	private $stackService;
 
 	/**
 	 * @param string $appName
 	 * @param IRequest $request
 	 * @param StackService $stackService
+	 * @param BoardService $boardService
```

Comment:
i'd remove the phpdoc directly since we have it as type hints now and it doesn't add additional information

Reply:
I'm all for removing redundant phpdocs altogether when it does not add any complementary value. I just figured I should limit the scope of changes in this PR. However, I can remove all redundant phpdocs in this PR as well. I just needed to know if it's okay and we're on the same page on this matter.

Reply:
Let's keep this separate then :)